doc/adr/0014-marconi-query-interface.rst

```diff
@@ -0,0 +1,39 @@
+ADR 14: Marconi Query Interface
+======================================================
+
+Date: 2022-10-27
+
+Author(s)
+---------
+
+Kayvan Kazeminejad <kayvan.kazeminejad@iohk.io>
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+Some DApps need to index and query a portion of the blockchain for a specific list of addresses. `Marconi <https://plutus-apps--763.org.readthedocs.build/en/763/adr/0004-marconi-initiative.html>`_ provides a general solution for indexing the blockchain data.  The query interface adds reporting capabilities on top of Marconi for a specific list of addresses for both Haskell and non-Haskell applications.
```

Comment:
Some do, but I feel like most don't. Also the specific list of addresses is something passed as a command line argument and the filter is invisible as far as the web interface goes. Why are we mentioning this here?